dist/src/main/java/io/camunda/zeebe/shared/security/IdentityAuthenticationManager.java

```diff
@@ -25,13 +27,15 @@
 public final class IdentityAuthenticationManager implements AuthenticationManager {
 
   private final Identity identity;
```

Comment:
💭 Same comment here regarding wrapping all the identity sdk call into the service

Reply:
See my answer [here](https://github.com/camunda/camunda/pull/24196#discussion_r1824102403).